Description: Use fixed dates for manuals.
 These make the build reproducible. I have picked the file modificatin dates of
 the corresponding source .tex files.
Author: Sascha Steinbiss <sascha@steinbiss.name>
--- a/docs/dynamite.tex
+++ b/docs/dynamite.tex
@@ -1,4 +1,5 @@
 \documentstyle{article}
+\pdftrailerid{}
 \begin{document}
 
 \newcommand{\programtext}[1]{{\tt #1}}
@@ -10,6 +11,7 @@
 Hinxton, Cambridge CB10 1SA,\\
 England.\\
 Email: birney@sanger.ac.uk}
+\date{18/6/2001}
 
 \maketitle
  
--- a/docs/wise2.tex
+++ b/docs/wise2.tex
@@ -1,5 +1,6 @@
 
 \documentclass{article}
+\pdftrailerid{}
 %\usepackage{epsfig}
 \usepackage{graphicx}
 
@@ -13,6 +14,7 @@
 Hinxton, Cambridge CB10 1SD,\\
 England.\\
 Email: birney@ebi.ac.uk}
+\date{18/6/2001}
 
 \maketitle
  
--- a/docs/wise2api.tex
+++ b/docs/wise2api.tex
@@ -1,4 +1,5 @@
 \documentstyle{article}
+\pdftrailerid{}
 \begin{document}
 
 \newcommand{\programtext}[1]{{\tt #1}}
@@ -10,6 +11,7 @@
 Hinxton, Cambridge CB10 1SA,\\
 England.\\
 Email: birney@sanger.ac.uk}
+\date{18/6/2001}
 
 \maketitle
  
